cli/README.md

```diff
@@ -6,22 +6,32 @@ EMAP star database and the PACS image system. Once a set of queues are
 populated the consumers can be started, updated and the system extractions
 stopped cleanly.
 
+## Prerequisites
+
+`PIXL CLI` requires Python version 3.10.
+
+The CLI requires a `pixl_config.yml` file in the current working directory. A [sample
+file](../pixl_config.yml.sample) is provided in the root of the repository. For local testing, we
+recommend running `pixl` from the [`./tests/`](./tests/) directory, which contains a mock
+`pixl_config.yml` file.
+
+Running the tests requires [docker](https://docs.docker.com/get-docker/) to be installed.
+
 ## Installation
 
-```bash
-pip install -e ../pixl_core/ .
-```
+We recommend installing in a project specific virtual environment created using a environment
+management tool such as [conda](https://docs.conda.io/en/latest/) or [virtualenv](https://virtualenv.pypa.io/en/latest/).
 
-## Test
+Then install in editable mode by running
 
 ```bash
-./tests/run-tests.sh
+pip install -e ../pixl_core/ .
 ```
 
 ## Usage
 
 > **Note**
-> Services must be started prior to using the CLI
+> The `rabbitmq`, `ehr-api` and `pacs-api` services must be started prior to using the CLI
```

Comment:
By the user? Isn't docker compose responsible for this?

Reply:
Yep, but this is just to provide a bit more detail about the context.
But could definitely add note, something like "this is typically handled by `docker compose`". But I guess in theory, you _could_ run the CLI manually, which will then fail because these services are missing.